pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -186,6 +186,7 @@
     private boolean preciseTopicPublishRateLimitingEnable;
     private boolean encryptionRequireOnProducer;
 
+    private boolean exposingBrokerEntryMetadataToClientEnabled;
```

Comment:
Why do we need to add ServerCnx? Is it able to get from the ServiceConfiguration directly?